CRM/Batch/BAO/Batch.php

```diff
@@ -576,8 +576,10 @@ public static function displayTotals($actual, $expected) {
    *   Associated array of batch ids.
    * @param string $exportFormat
    *   Export format.
+   * @param bool $downloadFile
+   *   Download export file?.
    */
-  public static function exportFinancialBatch($batchIds, $exportFormat) {
+  public static function exportFinancialBatch($batchIds, $exportFormat, $downloadFile) {
```

Comment:
Add a default value to `downloadFile` to keep compatibility with older versions?

Reply:
@mlutfy ```exportFinancialBatch(...) is used only in one place as I grep`ed in codebase
```
Monishs-MacBook-Pro:civicrm monish$ grep -irn "exportFinancialBatch" CRM api/
CRM/Batch/BAO/Batch.php:45:   * Not sure this is the best way to do this. Depends on how exportFinancialBatch() below gets called.
CRM/Batch/BAO/Batch.php:600:  public static function exportFinancialBatch($batchIds, $exportFormat, $downloadFile) {
CRM/Financial/Form/Export.php:178:    CRM_Batch_BAO_Batch::exportFinancialBatch($batchIds, $this->_exportFormat, $this->_downloadFile);
```

So I think its not required to provide a default value. What do you say?